helper/src/main/java/me/lucko/helper/item/ItemStackBuilder.java

```diff
@@ -63,7 +63,7 @@
     private final ItemStack itemStack;
 
     public static ItemStackBuilder of(Material material) {
-        return new ItemStackBuilder(new ItemStack(material)).hideAttributes();
+        return of(new ItemStack(material));
```

Comment:
What's the purpose of removing `#hideAttributes`? Anyone who accesses ItemStackBuilder through this method would have altered performance without any clear reasoning as to why the change has been applied.